integrationTests/consensus/consensusSigning_test.go

```diff
@@ -40,8 +40,9 @@ func initNodesWithTestSigner(
 	for shardID := range nodes {
 		if numInvalid < numNodes {
 			for i := uint32(0); i < numInvalid; i++ {
-				iCopy := i
-				nodes[shardID][i].MultiSigner.CreateSignatureShareCalled = func(privateKeyBytes, message []byte) ([]byte, error) {
+				ii := numNodes - i - 1
+				iCopy := ii
```

Comment:
iCopy is really needed? I think you can remove it and use directly ii in L47

Reply:
right, done